pl_examples/domain_templates/imagenet.py

```diff
@@ -121,7 +101,7 @@ def __accuracy(output, target, topk=(1,)):
 
             res = []
             for k in topk:
-                correct_k = correct[:k].view(-1).float().sum(0, keepdim=True)
+                correct_k = correct[:k].reshape(-1).float().sum(0, keepdim=True)
```

Comment:
A general question: Shouldn't we rather use the functional interface here since we already got this? @Borda @tchaton 

If we want to keep it like that, that's also fine with me :)

Reply:
Well, I think it looks great for an ImageNet beginner since the accuracy method is as the same as the official one.